build.sh

```diff
@@ -2,6 +2,11 @@
 
 set -e
 
-source $(dirname $0)/_main.sh "build publishToMavenLocal"
+source $(dirname $0)/_main.sh
+TEMP_PROJECT_VERSION="temp-version"
+
+runInBuilder "${GIT_COMMANDS}
```

Comment:
It's easy to forget about `GIT_COMMAND` in a new usage of `runInBuilder`
Can you run it internally always?